ts/components/DevicePairingDialog.tsx

```diff
@@ -59,23 +59,23 @@ export class DevicePairingDialog extends React.Component<Props, State> {
     const waitingForRequest = this.state.view === 'waitingForRequest';
     const nothingPaired = this.state.data.length === 0;
 
-    const renderPairedDevices = this.state.data.map((pubKey: any) => {
-      const pubKeyInfo = this.getPubkeyName(pubKey);
-      const isFinalItem =
-        this.state.data[this.state.data.length - 1] === pubKey;
-
-      return (
-        <div key={pubKey}>
-          <p>
-            {pubKeyInfo.deviceAlias}
-            <br />
-            <span className="text-subtle">Pairing Secret:</span>{' '}
-            {pubKeyInfo.secretWords}
-          </p>
-          {!isFinalItem ? <hr className="text-soft fullwidth" /> : null}
-        </div>
-      );
-    });
+    // const renderPairedDevices = this.state.data.map((pubKey: any) => {
+    //   const pubKeyInfo = this.getPubkeyName(pubKey);
+    //   const isFinalItem =
+    //     this.state.data[this.state.data.length - 1] === pubKey;
+
+    //   return (
+    //     <div key={pubKey}>
+    //       <p>
+    //         {pubKeyInfo.deviceAlias}
+    //         <br />
+    //         <span className="text-subtle">Pairing Secret:</span>{' '}
+    //         {pubKeyInfo.secretWords}
+    //       </p>
+    //       {!isFinalItem ? <hr className="text-soft fullwidth" /> : null}
+    //     </div>
+    //   );
+    // });
```

Comment:
Should we just remove those comments? Or we might need them later?

Reply:
Definitely need them. Just commented so lint doesn't complain.